### DISCUSSION TITLE
CMake Error: the following variables are used in this project, but they are set to NOTFOUND

### DISCUSSION THREAD

**Discussion by ALMEIDAESTIA** [Estia-1a/projetGenieInfo_public#90] [Q&A] [answered]
Bonjour, 
J’ai un problème car je n’arrive pas à faire l’exercice dimensions mais quand je fais « build » j’ai une erreur.
 
En effet il y a « CMake Generate step failed. Build files cannot be regenerated correctly »
 
Je ne sais pas comment corriger cette erreur.
![probleme projet info](https://github.com/Estia-1a/projetGenieInfo_public/assets/122285259/68366102-3626-4e11-84bb-91b0dc6220e3)

**Comment by adisdier** [comment, marked as answer]
Bonjour,

L'erreur t'indique que CMake n'a pas trouvé les bibliothèques estia-image et getopt. . Il faut qu'il y ait ces répertoires à la racine de ton projet. Les répertoires des librairies ne doivent pas contenir le numéro de la version.

Alexandre

**Comment by LucasCGD29** [comment]
Bonjour, 
J'ai un problème avec CMake. Voici le message d'erreur :
![Capture d’écran 2023-06-20 124653](https://github.com/Estia-1a/projetGenieInfo_public/assets/119681841/49e39528-f4bb-4198-b5e8-6b800959d1fa)
Il me dit que DependInfo.cmake file not found, alors qu'il est présent dans mon fichier freud.dir présent dans mon fichier build.
Je ne sais pas comment régler ce problème.
Merci d'avance.

**Reply by dhmmasson** [reply]
Est-ce que tu peux créer une question à part pour ton problème
